src/core/library.c

```diff
@@ -191,6 +191,10 @@ MsQuicLibraryInitialize(
     QUIC_STATUS Status = QUIC_STATUS_SUCCESS;
     BOOLEAN PlatformInitialized = FALSE;
     uint32_t DefaultMaxPartitionCount = QUIC_MAX_PARTITION_COUNT;
+    const QUIC_UDP_DATAPATH_CALLBACKS DatapathCallbacks = {
```

Comment:
Even if these are const, are they still stack allocated? shouldn't this be global?

Reply:
No need. The datapath copies the contents.